cypress/support/dropFile/index.js

```diff
@@ -15,3 +15,19 @@ Cypress.Commands.add("dropFile", { prevSubject: "element" }, (subject, fileName,
             });
         });
 });
+
+// drag and drop multiple files
+Cypress.Commands.add("upload_bulk_files", (selector, fileUrlArray, type = "") => {
```

Comment:
Nitpick: let's use camel case here, like we have for example for `cmsCreateContentModel`.